lib/oauthUtil.js

```diff
@@ -45,7 +46,9 @@ function loadPopup(src, options) {
 
 function getWellKnown(sdk) {
   // TODO: Use the issuer when known (usually from the id_token)
-  return http.get(sdk, sdk.options.url + '/.well-known/openid-configuration');
+  return http.get(sdk, sdk.options.url + '/.well-known/openid-configuration', {
+    responseCacheDuration: config.DEFAULT_CACHE_DURATION
```

Comment:
Related to other comment, but seems like it'd be nicer to make the a boolean "cacheResponse" and just load the time from the config. Or do you foresee having different cache timeouts for different requests?